File: testing/Python/NumPy/project_3.c
import numpy

l = [[[1,2], [3,4]], [[5,6],[7,8]], [[9,10], [11,12]]]
print(l)

l = numpy.array(l)
print(l)
print("numpy.size: ", numpy.size(l, axis=None))
print("numpy.shape: ", numpy.shape(l))
print("numpy.shape dimention 3: ", numpy.size(l, axis=2))


print("l[2]: ", l[2])
print("l[2][1]: ", l[2][1])
print("l[2][1][0]: ", l[2][1][0])

File: testing/Python/Pandas/project_3.py
import pandas as pd
import numpy as np

df = pd.DataFrame(np.random.randn(5,4), index=['a', 'b', 'c', 'd', 'e'], columns=['w', 'x', 'y', 'z'])

print(df)
print(df['w'])
#print(df['w','z'])
#print(df['w']['a'])
#print(df['a']['w'])

df = df.drop('z', axis=1)

print(df)
k = df['w'] + df['y']
print(k)
print(df.loc['a','w'])
print(df[df['w']>0]['y'])
